ext/pdo_odbc/tests/bug80783a.phpt

```diff
@@ -31,9 +30,9 @@ var_dump($data === $string);
 ?>
 --CLEAN--
 <?php
-require 'ext/pdo/tests/pdo_test.inc';
-$db = PDOTest::test_factory(dirname(__FILE__) . '/common.phpt');
-$db->exec("DROP TABLE IF EXISTS bug80783a");
+require_once __DIR__ . '/inc/odbc_pdo_test.inc';
+$db = ODBCPDOTest::factory();
+$db->exec("DROP TABLE IF EXISTS bug80783a_pdo_odbc");
```

Comment:
General comment on the pattern: I like removing the hardcoded table names from the query strings. It’s a little annoying that it has to be hardcoded here after it was unhardcoded for all the tests in their `FILE` sections, but I see why you probably didn’t do it for `CLEAN` (it’d be an ugly use of `ENV`/`ARGS`, and I’m not sure if those get propagated to `CLEAN` to begin with).